README.md

```diff
@@ -40,8 +40,28 @@ This is a portable library so you can use it anywhere.
 This library should be available on NuGet for easy consumption by your projects.
 You may also copy and paste the P/Invoke signatures you need directly into your projects if desired. 
 
+Packages:
+
+Library      | Package name     | NuGet       | Description
+-------------|------------------|-------------|-------------
+kernel32.dll |`PInvoke.Kernel32`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.Kernel32.svg)](https://www.nuget.org/packages/PInvoke.Kernel32)|Windows Kernel API
+user32.dll   |`PInvoke.User32`  | [![NuGet](https://img.shields.io/nuget/v/PInvoke.User32.svg)](https://www.nuget.org/packages/PInvoke.User32)|Windows User Interface
+gdi32.dll    |`PInvoke.Gdi32`   | [![NuGet](https://img.shields.io/nuget/v/PInvoke.Gdi32.svg)](https://www.nuget.org/packages/PInvoke.Gdi32)|[Windows Graphics Device Interface][Gdi]
+setupapi.dll |`PInvoke.SetupApi`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.SetupApi.svg)](https://www.nuget.org/packages/PInvoke.SetupApi)|[Windows application programming interface][SetupApi]
```

Comment:
Shouldn't this one's description be "Windows *setup* application programming interface"?
And you use the API acronym for kernel32. How about we use it here as well?